code/modules/paperwork/contract.dm

```diff
@@ -273,14 +273,16 @@
 	if(!user.dna)
 		return -1
 	user.dna.add_mutation(HULK)
+	user.dna.add_mutation(TK)
 	var/obj/item/organ/regenerative_core/organ = new /obj/item/organ/regenerative_core
 	organ.Insert(user)
+	ADD_TRAIT(user, TRAIT_MUTE, TRAIT_DEVILTRAIT)//makes you mute(mime meta)
```

Comment:
This is actually an UPSIDE since it lets you silently bust through walls with Hulk.

Reply:
it still makes wall breaking sounds you just dont yell, and I am pretty sure if you want to get around the yelling into radio thing you can just take off your headset, but let me check

Reply:
yeah this is the case